README.md

```diff
@@ -1,2 +1,17 @@
-# thermal_base
-Decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+# Base codes for Thermography
+
+A python package for decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+
+## Install
+Install this package with `pip install git+ssh://git@github.com/detecttechnologies/thermal_base.git@main`
```

Comment:
Will using the `ssh://` require the user to have an SSH key-pair that's registered to a github account? If yes, can you look for an alternative?

Reply:
Fixed